### PR TITLE
Add backpack

### PR DESCRIPTION
**Pull request by nick-youngblut**

### CODE REVIEW

File: recipes/backpack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (isuruf) [inline recipes/backpack/meta.yaml]: Can this be `noarch: python`?

**Comment by nick-youngblut** [comment]
@isuruf I'm not sure how to deal with the current error:

```
conda_build.exceptions.CondaBuildException: Empty package; python present in build and host deps.  You probably picked up the build environment's python  executable.  You need to alter your recipe to  use the PYTHON env var in your recipe to run that executable.
```

The conda-forge docs on `noarch` could use more info on how to properly create a noarch recipe (or at least some good examples).

File: recipes/backpack/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
```
Comment (isuruf) [inline recipes/backpack/meta.yaml]: Change this to host

Comment (isuruf) [reply recipes/backpack/meta.yaml]: Move pip to host

**Comment by isuruf** [comment]
Looks like it is compiling extensions. You should remove `noarch: python` and add compilers.

File: recipes/backpack/meta.yaml [hunk 3]
```diff
@@ -14,13 +14,11 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  noarch: python
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
```
Comment (isuruf) [inline recipes/backpack/meta.yaml]: Add compilers here

**Comment by nick-youngblut** [comment]
@isuruf I'm not sure what compilers to add. `conda skeleton pypi backpack` doesn't add any compilers to the meta.yaml file. Also, I believe that the recipe passes all checks without compilers (before adding `noarch`)

**Comment by bgruening** [comment]
Thanks all!

File: recipes/backpack/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: Useful utilities for Python.
+  description: "Useful utilities for Python. Supports Python 2.7+ and 3.2+."
+  doc_url: https://github.com/sdispater/backpack
+  dev_url: https://github.com/sdispater/backpack
+
+extra:
+  recipe-maintainers:
```
Comment (jdblischak) [inline recipes/backpack/meta.yaml]: Why were the R maintainers added to this Python recipe? Is it a dependency for an R package?

**Pull request merged**